testData/highlighting/equalinif.go

```diff
@@ -0,0 +1,29 @@
+package equalinif
+
+func <warning descr="Unused function 'eqlfunc'">eqlfunc</warning>() {
```

Comment:
Just replace `<warning descr="Unused function 'eqlfunc'">eqlfunc</warning>` in your test data with `_`. There're lots of examples around, please have a look.